packages/lib/gpcircuits/test/proto-pod-gpc.spec.ts

```diff
@@ -117,6 +134,20 @@ const sampleInput: ProtoPODGPCInputs = {
   ],
   /*PUB*/ entryIsValueEnabled: 59n,
   /*PUB*/ entryIsValueHashRevealed: 21n,
+
+  // Entry constraint modules.
```

Comment:
Why did this move?

Reply:
Still wondering why Entry constraint module inputs got moved to be in the middle of Entry module inputs.  These are intended to be grouped by module, not by name, as per the comments.

Reply:
Fixed.